client/auth/style.scss

```diff
@@ -2,22 +2,26 @@
  * Oauth login page, used in WordPress Desktop App
  */
 
-html.is-desktop {
-	.logged-out-auth {
-		background: $blue-wordpress;
-
-		.wp-content {
-			margin: 0 auto;
-			overflow: hidden;
-			max-width: none;
-			padding-left: 0;
-			padding-right: 0;
-		}
+.is-section-login {
+	background: $blue-wordpress;
+	height: 100%;
```

Comment:
> This block obviates most of these styles. They should be removed once this is merged.

I created an issue for that here: https://github.com/Automattic/wp-desktop/issues/67